store/tikv/oracle/oracles/pd.go

```diff
@@ -45,9 +45,10 @@ func NewPdOracle(pdClient pd.Client, updateInterval time.Duration) (oracle.Oracl
 		c:    pdClient,
 		quit: make(chan struct{}),
 	}
-	go o.updateTS(updateInterval)
+	ctx := context.TODO()
```

Comment:
It seems we share a `ctx` in this struct, can we make the `ctx` to a field of `pdOracle` @hhkbp2